applications/asset_tracker/src/main.c

```diff
@@ -1388,6 +1388,32 @@ void on_pairing_done(void)
 	}
 }
 
+void fota_done_handler(const struct cloud_event *const evt)
+{
+#if defined(CONFIG_NRF_CLOUD_FOTA)
+	enum nrf_cloud_fota_type fota_type = NRF_CLOUD_FOTA_TYPE__INVALID;
+
+	if (evt && evt->data.msg.buf) {
+		fota_type = *(enum nrf_cloud_fota_type *)evt->data.msg.buf;
+		LOG_INF("FOTA type: %d", fota_type);
+	}
+
+	if (fota_type == NRF_CLOUD_FOTA_MODEM) {
```

Comment:
What is the purpose of this if, there is not a single instruction in the block? Perhaps the functionality from the comment should this be demonstrated here?

Reply:
as currently implemented, asset tracker allows nrf_cloud_fota to handle the modem update by rebooting.
i can just remove this if block if you don't feel it is useful.

Reply:
Well, IMO a place for documentation of nrf_cloud functionalities is in https://github.com/nrfconnect/sdk-nrf/blob/master/include/net/nrf_cloud.rst, not in an empty `if` created just for adding this comment. But let's hear also from others first, no strong push here from my side, perhaps it could stay like this.

Reply:
it looks like there is a new sample `samples/nrf9160/http_update/full_modem_update` which does the re-init after update.  
that sample can simply call shutdown and then init, since it manually initializes the modem lib.  
doing the re-init in asset tracker would be much more involved since `CONFIG_NRF_MODEM_LIB_SYS_INIT=y` 
i will remove the empty `if` block and comment.